inst/qml/msaGaugeRR.qml

```diff
@@ -532,4 +532,164 @@ Form
 			}
 		}
 	}
+
```

Comment:
I assume this is the same for all these analyses, so it would be best to place it in a common qml file and import it with parameters, if needed